certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+          },
+          "400": {
+            "description": "If the item is not valid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "If user is not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "If operation is not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Fields/{entity}": {
+      "get": {
+        "tags": [
+          "Fields"
+        ],
+        "summary": "Get fields",
+        "description": "Gets fields information for specified entity(e.g. Project, Idea, Resource etc)",
+        "operationId": "Fields_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "entity",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Entity",
+            "description": "Entity name",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "important",
+            "enum": [
+              "Project",
+              "Program",
+              "Portfolio",
+              "Resource",
+              "KeyDate",
+              "Challenge",
+              "Idea",
+              "Task",
+              "Roadmap",
+              "Issue"
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns entity field infos",
+            "schema": {
+              "$ref": "#/definitions/FieldResponseList"
+            }
+          },
+          "400": {
+            "description": "If the entity not supported",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "If operation is not allowed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Ideas/{id}": {
+      "get": {
+        "tags": [
+          "Ideas"
+        ],
+        "summary": "Get an Idea",
+        "description": "Gets an existing Idea by Id",
+        "operationId": "Ideas_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/idea_id__in_path"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns existing item",
+            "schema": {
+              "$ref": "#/definitions/IdeaResponseResponse"
+            },
+            "examples": {

```

Comment:
@vsorokin-ppmexpress can you please remove anything that looks like real ID/GUID and replace it with test value, e.g. all the same numbers/letters? Can you please do it for all examples in all operations?